stream/address_extractor.js

```diff
@@ -61,9 +61,15 @@ module.exports = function(){
             peliasLogger.debug('[address_extractor] found multiple house numbers: ', streetnumbers);
           }
 
+          // If unit is set to an address add it to default name since streetno and street will be the same
+          var unit = '';
+          if(doc.address_parts.hasOwnProperty('unit')) {
+            unit = ' ' + doc.address_parts.unit;
+          }
+
           // copy data to new document
           record = new Document( 'openstreetmap', 'address', newid.join(':') )
-            .setName( 'default', streetno + ' ' + doc.address_parts.street )
+            .setName( 'default',  streetno + unit + ' ' + doc.address_parts.street)
```

Comment:
Hey @sweco-semhul,
Similarly to the [OA importer](https://github.com/pelias/pelias/issues/618#issuecomment-342986871), what do you think about changing this to streetno + street + unit?